main.go

```diff
@@ -205,6 +205,9 @@ func processMessage(body []byte, routingKey, contentType string, headers amqp.Ta
 	}
 
 	for _, msg := range msgs {
+		if config.SupressHighlightsByDoc {
```

Comment:
move this into the `outputMessage` function which is called right before the JSON/LEEF formatting